.evergreen/serverless/create-instance.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+set -o errexit
+set +o xtrace # disable xtrace to ensure credentials aren't leaked
+
+if [ -z "$PROJECT" ]; then
+    echo "Project name must be provided via PROJECT environment variable"
+    exit 1
+fi
+INSTANCE_NAME="$RANDOM-$PROJECT"
```

Comment:
This is a pretty simple / crude way to ensure the instances have unique names, but it's effective and works within the relatively strict limitations that are set on cluster names: https://docs.atlas.mongodb.com/reference/atlas-limits/#label-limits

Reply:
Oh clever, TIL about `$RANDOM` (and `$SECONDS`).